test/integration/alertsMenuView_integrationTests.js

```diff
@@ -13,11 +13,49 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 /*global sinon, todo, chai, describe, it, calledOnce*/
 //integration tests
-
 describe('alertsMenuView.js spec', function() {
+    var now_date = new Date();
```

Comment:
Lines 19 through 58 should be inside the `beforeEach` block. I would recommend moving it to line 128.
For this test, it doesn't exactly matter. But this is good practice because for a test where you want something like `new Date()` to be refreshed for every test, that will only happen inside the beforeEach block.